src/modules/auto_leave_raids/index.js

```diff
@@ -0,0 +1,45 @@
+import $ from 'jquery';
```

Comment:
Night is looking into removing jquery, can we try converting this file to vanilla js?